src/patternfly/components/Skeleton/skeleton.scss

```diff
@@ -0,0 +1,178 @@
+
+
+  // Text modifiers
+  --pf-c-skeleton__element--m-text-4xl--Height: calc(var(--pf-global--FontSize--4xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-3xl--Height: calc(var(--pf-global--FontSize--3xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-2xl--Height: calc(var(--pf-global--FontSize--2xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-xl--Height: calc(var(--pf-global--FontSize--xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-lg--Height: calc(var(--pf-global--FontSize--lg) * var(--pf-global--LineHeight--md));
+  --pf-c-skeleton__element--m-text-md--Height: calc(var(--pf-global--FontSize--md) * var(--pf-global--LineHeight--md));
+  --pf-c-skeleton__element--m-text-sm--Height: calc(var(--pf-global--FontSize--sm) * var(--pf-global--LineHeight--md));
+
+  // Width
+  --pf-c-skeleton__element--m-width-25--Width: 25%;
+  --pf-c-skeleton__element--m-width-33--Width: calc(100% / 3);
+  --pf-c-skeleton__element--m-width-50--Width: 50%;
+  --pf-c-skeleton__element--m-width-66--Width: calc(100% / 3 * 2);
+  --pf-c-skeleton__element--m-width-75--Width: 75%;
+}
+
+.pf-c-skeleton__element {
+  position: relative;
+  height: var(--pf-c-skeleton__element--Height);
+  overflow: hidden;
+  background-color: var(--pf-c-skeleton__element--BackgroundColor);
+  border-radius: var(--pf-c-skeleton__element--BorderRadius);
+
+  &::before {
+    display: block;
+    padding-bottom: 0;
+    content: "";
+  }
+
+  &::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: var(--pf-c-skeleton__element--after--ZIndex);
+    display: block;
+    width: 100%;
+    height: 100%;
+    content: "";
+    background: linear-gradient(var(--pf-c-skeleton__element--after--LinearGradientAngle), var(--pf-c-skeleton__element--after--LinearGradientColorStop1), var(--pf-c-skeleton__element--after--LinearGradientColorStop2), var(--pf-c-skeleton__element--after--LinearGradientColorStop3));
+    transform: translateX(var(--pf-c-skeleton__element--after--TranslateX));
+    animation: var(--pf-c-skeleton__element--after--AnimationName) var(--pf-c-skeleton__element--after--AnimationDuration) var(--pf-c-skeleton__element--after--AnimationTimingFunction) var(--pf-c-skeleton__element--after--AnimationDelay) var(--pf-c-skeleton__element--after--AnimationIterationCount);
+  }
+
+  // Circle
+  &.pf-m-circle {
+    --pf-c-skeleton__element--BorderRadius: var(--pf-c-skeleton__element--m-circle--BorderRadius);
+  }
+
+  // Width
+  &.pf-m-width-sm {
+    width: var(--pf-c-skeleton__element--m-width-sm--Width);
+  }
+
+  &.pf-m-width-md {
+    width: var(--pf-c-skeleton__element--m-width-md--Width);
+  }
+
+  &.pf-m-width-lg {
+    width: var(--pf-c-skeleton__element--m-width-lg--Width);
+  }
+
+  &.pf-m-width-sm,
+  &.pf-m-width-md,
+  &.pf-m-width-lg {
+    --pf-c-skeleton__element--Height: none;
```

Comment:
Another way of doing this would be to let width set width, and let height set height, and have `pf-m-circle` and `pf-m-square` unset the height and use `padding-bottom: 100%`

Reply:
that would mean having the modifier `pf-m-square` though.. not sure we need to add that